src/core/pipeline/pipeline.ts

```diff
@@ -1,7 +1,15 @@
-import { Stage } from './stages';
+import {
+  GroupStage,
+  LimitStage,
+  MatchStage,
+  Stage,
+  StageDefinition,
+} from './stages';
 import { Document } from '../index';
 
 export interface Pipeline {
+  db?: string;
+  collection?: string;
```

Comment:
Why can it be undefined if it is required in the createPipeline function and always return db and collection ?

Reply:
Same than previous, for testing. I removed the '?' and I adapted the tests